docs/partners/plan-provisioning-direct-api.md

```diff
@@ -0,0 +1,151 @@
+# Provisioning and Cancelling Jetpack Plans – Querying the API Directly
+
+In [another document](plan-provisioning.md), we discussed how to provision and cancel plans by using the shell script that ships with Jetpack. But, for partners where running shell commands won't work, it is possible to communicate directly to the API on WordPress.com.
+
+### Getting a Jetpack Partner access token
+
+When you become a Jetpack partner, we will provide you with your partner ID and a secret key. Typically you just pass these values directly in to the `bin/partner-provision.sh` and `bin/partner_cancel.sh` scripts. But, when calling the WordPress.com API directly, you'll first need to get an access token with for your partner ID with a scope of `jetpack-partner`.
+
+To do that, you'll make a `POST` request to the `https://public-api.wordpress.com/oauth2/token` endpoint passing with the request parameters mentioned below.
+
+#### Endpoint Information
+
+- __Method__: POST
+- __URL__:    https://public-api.wordpress.com/oauth2/token
+
+#### Request Parameters
+
+- __grant_type__:    Value should be `client_credentials`
+- __scope__:         Value should be `jetpack-partner`
+- __client_id__:     The partner ID that we provide you
+- __client_secret__: The partner secret that we provide you
+
+#### Response Parameters
+
+- __access_token__: (string) This is the access token we'll need for the API calls below.
+- __token_type__:   (string) This should be `bearer`.
+- __blog_id__:      (int) This should be `0`.
+- __blog_url__:     (int) This should be `0`.
+- __scope__:        (string) This should be `jetpack-partner`.
+
+Note: You only need to create the `access_token` once.
+
+#### Examples
+
+Here is an example using cURL in shell.
+
+```shell
+curl --request POST \
+    --url https://public-api.wordpress.com/oauth2/token \
+    --header 'cache-control: no-cache' \
+    --header 'content-type: multipart/form-data;' \
+    --form client_id={PARTNER_ID} \
```

Comment:
Is this intended to be shell-interpolated? If so, it should probably be written as `${PARTNER_ID}` or `$PARTNER_ID`

Reply:
Nope. I just meant it as a placeholder so the partner placed their ID there.